logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/ServerConfig.kt

```diff
@@ -15,7 +15,8 @@ data class ServerConfig(
     val blackListUrl: String,
     val teamsUrl: String,
     val websiteUrl: String,
-    val title: String
+    val title: String,
+    val androidSenderId: String?
```

Comment:
Why is it called `androidSenderId` ?

What happens if I register a client on JVM, for example on the `cli` module?